charmcraft/templates/init-machine/tests/spread/lib/cloud-config.yaml.j2

```diff
@@ -0,0 +1,10 @@
+#cloud-config
+
+ssh_pwauth: true
+
+users:
+  - default
+  - name: spread
+    plain_text_passwd: spread
```

Comment:
Is there a way to do this using spread's generated passwords?

If you don't have a simple way to do it immediately, please make an issue so we don't forget about it - I don't want `spread/spread` to become one of the username/password pairs bad guys start scanning cloud services with :-)